courses/machine_learning/deepdive2/time_series_prediction/solutions/3_modeling_bqml.ipynb

```diff
@@ -510,8 +510,8 @@
     }
```

Comment:
<p>I think the "# TODO 2a" will produce an error. Better use "--" for comments in a BQ cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>